currencyConverter.js

```diff
@@ -0,0 +1,66 @@
+var request = require('request');
+var cheerio = require('cheerio');
+
+var roundOff = function(currency){
+
+	/**
+	* Rounds off currency output 2 decimal places
+	*/
+
+	return Math.round(currency * 100) / 100;
+
+}
+
+var getCurrency = function(param, param1){
```

Comment:
The parameter names need to be better. I suggest something like `from` and `to`.